target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -72,6 +72,17 @@ define Device/ubnt-xm
   KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma | uImage lzma
 endef
 
+define Device/ubnt-sw
+  $(Device/ubnt)
+  DEVICE_VARIANT := SW
+  DEVICE_PACKAGES += kmod-usb-ohci
+  IMAGE_SIZE := 7552k
+  UBNT_TYPE := SW
+  UBNT_CHIP := ar7240
+  ATH_SOC := ar7242
```

Comment:
Please move this to the top (below $(Device/ubnt))

Reply:
I've adjusted this to match the format seen in the patch of the previous  review